platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/collections/LongList.java

```diff
@@ -169,7 +160,11 @@ protected LongList(final FileChannel fileChannel) throws IOException {
             final ByteBuffer headerBuffer = readFromFileChannel(fileChannel, formatMetadataSize);
             numLongsPerChunk = headerBuffer.getInt();
             memoryChunkSize = numLongsPerChunk * Long.BYTES;
-            maxIndexThatCanBeStored.set(headerBuffer.getLong());
+            if (formatVersion == INITIAL_VERSION) {
```

Comment:
You could move the comment up and collapse this if to single-line for simplicity, if you prefer.

Reply:
will do